.github/workflows/cd.main.yml

```diff
@@ -0,0 +1,59 @@
+name: CD
+on:
+  push:
+    branches:
+      - main
+env:
+  ORGANIZATION_SLUG: templates
+  INSTRUQT_TOKEN: ${{ secrets.INSTRUQT_TOKEN }}
+jobs:
+  deploy-changes:
+    name: Deploy changes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install CLI
+        run: |
+          curl https://github.com/instruqt/cli/releases/download/1909-0b7cb45/instruqt-linux-1909-0b7cb45.zip -L -o cli.zip
+          unzip cli.zip
+          mv ./instruqt /usr/local/bin
+          rm cli.zip
+          instruqt update
+          instruqt config set organization ${{ env.ORGANIZATION_SLUG }}
+      - name: Checkout template repository
+        uses: actions/checkout@v2
+      - name: Get changed files
+        id: scope
+        uses: jitterbit/get-changed-files@v1
+      - name: Finding modified tracks
```

Comment:
I don't think it is necessary to only push changed tracks. We can just push all tracks, since the repository is leading anyway. This simplifies the action and makes it less prone to bugs were the change detection is misbehaving.

Reply:
I would expect `last_edited_at` to stay the same when pushing a track that doesn't have changes, is this not the case?

Reply:
https://github.com/instruqt/gozilla/pull/567

This change should fix the redundancy. With this change, we can just push all tracks.